exec/java-exec/src/test/java/org/apache/drill/exec/expr/fn/registry/FunctionRegistryHolderTest.java

```diff
@@ -202,26 +250,38 @@ public void testGetAllFunctionsWithSignatures() {
   public void testGetHoldersByFunctionNameWithVersion() {
     List<DrillFuncHolder> expectedResult = newJars.values().stream()
         .flatMap(Collection::stream)
-        .filter(f -> "lower".equals(f.getName()))
+        .filter(f -> LOWER_FUNC_NAME.equals(f.getName()))
         .map(FunctionHolder::getHolder)
         .collect(Collectors.toList());
 
     assertFalse(expectedResult.isEmpty());
     AtomicInteger version = new AtomicInteger();
-    compareTwoLists(expectedResult, registryHolder.getHoldersByFunctionName("lower", version));
+    compareTwoLists(expectedResult, registryHolder.getHoldersByFunctionName(LOWER_FUNC_NAME, version));
     assertEquals("Version number should match", version.get(), registryHolder.getVersion());
   }
 
   @Test
   public void testGetHoldersByFunctionName() {
-    List<DrillFuncHolder> expectedResult = newJars.values().stream()
-        .flatMap(Collection::stream)
-        .filter(f -> "lower".equals(f.getName()))
-        .map(FunctionHolder::getHolder)
-        .collect(Collectors.toList());
+    List<DrillFuncHolder> expectedUniqueResult = Lists.newArrayList();
```

Comment:
Please use `new ArrayList<>` here and below.

Reply:
Done. removed all references to Lists and removed the Guava import as well.